cmd/grr/main.go

```diff
@@ -81,22 +81,23 @@ func main() {
 }
 
 func createRegistry(context *config.Context) grizzly.Registry {
-	providerInitFuncs := []func() (grizzly.Provider, error){
-		func() (grizzly.Provider, error) { return grafana.NewProvider(&context.Grafana) },
-		func() (grizzly.Provider, error) { return mimir.NewProvider(&context.Mimir) },
-		func() (grizzly.Provider, error) { return syntheticmonitoring.NewProvider(&context.SyntheticMonitoring) },
+	providerInitFuncs := []func() grizzly.Provider{
+		func() grizzly.Provider { return grafana.NewProvider(&context.Grafana) },
+		func() grizzly.Provider { return mimir.NewProvider(&context.Mimir) },
+		func() grizzly.Provider { return syntheticmonitoring.NewProvider(&context.SyntheticMonitoring) },
 	}
```

Comment:
```suggestion
	providers := []grizzly.Provider{
		grafana.NewProvider(&context.Grafana),
		mimir.NewProvider(&context.Mimir),
		syntheticmonitoring.NewProvider(&context.SyntheticMonitoring),
	}
```
Doesn't need to be a func if we're validating in a separate func